bioneuron_oracle/builder.py

```diff
@@ -265,7 +248,8 @@ def build_connection(model, conn):
                                              transform=transform,
                                              weights=syn_weights)
 
-    if conn_out_bioneuron or conn_out_bioneuron_slice:
+    elif (isinstance(conn_pre, nengo.Ensemble) and
+          isinstance(conn_pre.neuron_type, BahlNeuron)):
```

Comment:
Notice because this was `if` rather than `elif` before, you were building a whole host of signals and operators that end up having no influence any time you were connecting into a bio ensemble. Before this commit, the 20 tests took 1000 seconds on my computer, and after this commit the tests take 500 seconds! 😀